<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>282</width>
      <height>442</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,2,3,4,5">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>种子：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="seed">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>40</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string>矩阵：</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="matrix">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>280</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>280</height>
        </size>
       </property>
       <property name="toolTipDuration">
        <number>-2</number>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::CustomDashLine</enum>
       </property>
       <property name="rowCount">
        <number>7</number>
       </property>
       <property name="columnCount">
        <number>7</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="verticalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>5</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>6</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>7</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>1</string>
        </property>
        <property name="font">
         <font>
          <family>Script MT Bold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>3</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>4</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>5</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>6</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>7</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>精度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="precision">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>偏移量：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="offset">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="calculate">
       <property name="text">
        <string>计算</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>0</y>
      <width>301</width>
      <height>439</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="samp1">
       <property name="text">
        <string>samp1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="samp2">
       <property name="text">
        <string>samp2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="samp3">
       <property name="text">
        <string>samp3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="samp4">
       <property name="text">
        <string>samp4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="samp5">
       <property name="text">
        <string>samp5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>计算方式：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button1">
       <property name="text">
        <string>幂法求最大特征值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button2">
       <property name="text">
        <string>反幂法求最小特征值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button3">
       <property name="text">
        <string>反幂法的原点平移法</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>计算结果(特征值λ)：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="res0">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>计算结果(特征向量)：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="res1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>40</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>440</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
